tcf_website/management/commands/other_schools_department_fix.py

```diff
@@ -0,0 +1,76 @@
+from django.core.management.base import BaseCommand, CommandError
+from tcf_website.models import *
+
+
+class Command(BaseCommand):
+
+    help = 'Promotes subdepartments in \'Other Schools at the University of Virginia\' to departments'
+
+    def add_arguments(self, parser):
+        # Named (optional) arguments
+        parser.add_argument(
+            '--verbose',
+            action='store_true',
+            help='Verbose output',
+        )
+
+    def handle(self, *args, **options):
+
+        self.verbose = options['verbose']
+
+        # Take Other Schools at UVA and promote all subdepartments into departments
+        # Excluded schoosl not to split up
+        excluded = [
+            "College of Arts & Sciences",
+            "School of Engineering & Applied Science",
+            "UNKNOWN"]
+
+        if self.verbose:
+            dash = '-' * 185
+            print(dash)
+            print(
+                '{:<70s}{:<70s}{:<70s}'.format(
+                    'Subdepartment',
+                    'Department',
+                    'School'))
+            print(dash)
+
+        # Go through all schools that are not in the list of excluded
+        for school in School.objects.exclude(name__in=excluded):
+            # Go through all departments
+            for department in school.department_set.all():
+                # Go through all subdepartments
+                for subdepartment in department.subdepartment_set.all():
+                    if self.verbose:
+                        print(
+                            '{:<70s}{:<70s}{:<70s}'.format(
+                                subdepartment.name,
+                                department.name,
+                                school.name))
+                        print(dash)
+                    # Promote the subdepartment to a department
+                    new_department = Department(
+                        name=subdepartment.name,
+                        school=school
+                    )
```

Comment:
I don't believe that any of the sub/departments currently have any of the information filled in but I can copy over the description just in case? May be something we want to address later.

Reply:
If we're going for consistency then maybe `website` should be copied as well? Might be even more unnecessary considering that I can't even think of where we'd use a `website` field, but better safe than sorry?

Reply:
Maybe copy `website` as well? Might be completely unnecessary if we don't store the fields, but idk ¯\_(ツ)_/¯

Reply:
ok i don't understand why my comments on an edit with description being copied are being added to this code snippet which doesn't feature it, but @jackliu612 hopefully you get the message

Reply:
Subdepartments don't have a website field to copy over so I think it might be fine as it is now

Reply:
cool, all good then